lib/zinnia.js

```diff
@@ -283,6 +283,13 @@ async function handleEvents ({
     try {
       const event = JSON.parse(line)
       switch (event.type) {
+        case 'activity:started':
+          onActivity({
+            type: 'info',
+            message: `${capitalize(module)} started`,
```

Comment:
Do you think it's worth writing a new test for this? Any suggestions one how to write it?

Maybe this is already covered by the existing test here:

https://github.com/filecoin-station/core/blob/f77317966725f110151bda2edb443c0186aa0919/test/station.js#L15-L16

Thoughts?

Reply:
The tests are failing when I comment out the `onActivity` call - I think we are good here.

```js
        case 'activity:started':
          // onActivity({
          //   type: 'info',
          //   message: `${capitalize(module)} started`,
          //   source: module
          // })
          break
```

Test output:

```
  8 passing (48s)
  3 failing

  1) Station
       runs Zinnia:
     Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/bajtos/src/filstation/core/test/station.js)
      at listOnTimeout (node:internal/timers:573:17)
      at process.processTimers (node:internal/timers:514:7)

  2) Station
       outputs events:
     Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/bajtos/src/filstation/core/test/station.js)
      at listOnTimeout (node:internal/timers:573:17)
      at process.processTimers (node:internal/timers:514:7)

  3) Station
       outputs events json:
     Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/bajtos/src/filstation/core/test/station.js)
      at listOnTimeout (node:internal/timers:573:17)
      at process.processTimers (node:internal/timers:514:7)
```

Reply:
sounds good as it is yeah!